File: main/src/androidTest/java/cgeo/geocaching/connector/IConnectorTest.java
package cgeo.geocaching.connector;

import org.junit.Test;
import static org.assertj.core.api.Java6Assertions.assertThat;

public class IConnectorTest {

    @Test
    public void testName() {
        for (final IConnector connector : ConnectorFactory.getConnectors()) {
            if (connector != ConnectorFactory.UNKNOWN_CONNECTOR) {
                assertThat(connector.getName()).isNotEmpty().isNotBlank();
                assertThat(connector.getNameAbbreviated()).isNotEmpty().isNotBlank();
            }
        }
    }
}

File: main-kotlin/src/androidTest/kotlin/cgeo/geocaching/connector/IConnectorTest.kt
// Auto-converted from Java to Kotlin
// WARNING: This code requires manual review and likely has compilation errors
// Please review and fix:
// - Method signatures (parameter types, return types)
// - Field declarations without initialization
// - Static members (use companion object)
// - Try-catch-finally blocks
// - Generics syntax
// - Constructors
// - And more...

package cgeo.geocaching.connector

import org.junit.Test
import org.assertj.core.api.Java6Assertions.assertThat

class IConnectorTest {

    @Test
    public Unit testName() {
        for (final IConnector connector : ConnectorFactory.getConnectors()) {
            if (connector != ConnectorFactory.UNKNOWN_CONNECTOR) {
                assertThat(connector.getName()).isNotEmpty().isNotBlank()
                assertThat(connector.getNameAbbreviated()).isNotEmpty().isNotBlank()
            }
        }
    }
}
